.storybook/preview-head.html

```diff
@@ -0,0 +1,41 @@
+<style>
```

Comment:
Apercu web fonts are not free, so we shouldn't commit these files to a public npm package. Therefore no need include the fonts in this repository, we can apply the fonts in the website repository instead.

Reply:
Ok, I can do that. So when we view the storybook link we won't see how the fonts look at all, or is there another way to preview? I know there is also the playground now

Reply:
Sapera Brand will show the look of the fonts.

Reply:
this commit should be removed from history (using `rebase -i` otherwise the files would always be accessible and available in the git history)

Reply:
https://stackoverflow.com/a/52643437/1769515

Reply:
Great, thanks. I wanted to ask you @a14m about how to remove from the commit history actually.

Reply:
it's actually pretty dangerous and I messed the whole branch doing this... and ended up redoing the PR from the local backup in #10 (but they are not in the history anymore)